apps/src/weblab/WebLab.js

```diff
@@ -355,18 +354,10 @@ WebLab.prototype.onFinish = function(submit) {
   }
 };
 
-WebLab.prototype.getMaxProjectCapacity = function() {
-  return getStore().getState().maxProjectCapacity;
-};
-
-WebLab.prototype.setProjectSize = function(bytes) {
-  getStore().dispatch(actions.changeProjectSize(bytes));
-};
```

Comment:
functionality has moved to CdoBramble:
https://github.com/code-dot-org/code-dot-org/blob/b1d7492b5eb13541022bcb5a5e4f0412da763910/apps/src/weblab/CdoBramble.js#L234-L237